playground/ssr-conditions/__tests__/ssr-conditions.spec.ts

```diff
@@ -11,10 +11,14 @@ test('ssr.resolve.conditions affect non-externalized imports during ssr', async
   )
 })
 
-test('ssr.resolve.externalConditions affect externalized imports during ssr', async () => {
-  await page.goto(url)
-  expect(await page.textContent('.external-react-server')).toMatch('edge.js')
-})
+// externalConditions is only used for dev
+test.runIf(isServe)(
+  'ssr.resolve.externalConditions affect externalized imports during ssr',
+  async () => {
+    await page.goto(url)
+    expect(await page.textContent('.external-react-server')).toMatch('edge.js')
+  },
+)
 
 test('ssr.resolve settings do not affect non-ssr imports', async () => {
```

Comment:
Is the 3rd test here also not functioning for build? I was confused as `withRetry` is skipped during build.
Maybe better to writing `test.runIf(isServe)` explicitly?

Reply:
Ah, that's true. I added the third argument to make them run in build as I think we should test them in build.